### DISCUSSION TITLE
Motion Detection .PGM mask

### DISCUSSION THREAD

**Discussion by Solarek68** [gopro/labs#831] [Q&A]
Hi, I'm trying to use my own .PGM mask when running GoPro11 in video motion detection mode. Unfortunately, even though I upload the file to the MISC directory, the camera does not load the mask. What am I doing wrong? I'm preparing according to the instructions.

**Comment by dnewman-gpsw** [comment]
I just re-tested on HERO11 and it works. So you likely have a formatting issue.  This mask the right side of the image
[mask.zip](https://github.com/gopro/labs/files/14826275/mask.zip)  I tested with 4K 16:9.  The mask size suggested might be wrong in the instructions, this is a very old feature.  I just used the generated mask, and modified that.

**Reply by Solarek68** [reply]
Thank you!!! It is working. You are right. the problem was mask size.

**Comment by Fougl** [comment]
Hi, I hope those mask sizes and everything is updated correctly. I am using this feature, it is functioning good enough, as intented. It is an incredible feature. I hope it does not get forgotten
